web-app/src/main.js

```diff
@@ -1,11 +1,11 @@
 import { createApp } from 'vue';
 import '@/sw/register';
 import router from '@/router';
-import store from '@/store';
+// import store from '@/store';
```

Comment:
Keeping the store around for a bit - at this point just experimenting with composition API